package.json

```diff
@@ -40,7 +39,6 @@
     "web3-typescript-typings": "^0.7.2"
   },
   "dependencies": {
-    "bignumber.js": "^5.0.0",
```

Comment:
@krzkaczor no needed? It's already in `devDependencies` and `peerDependencies`

Reply:
Hmm yeah it should not be both `peerDependecies` and `dependencies`.